src/explorer.ts

```diff
@@ -198,22 +198,25 @@ export class OpenShiftExplorer implements TreeDataProvider<ExplorerItem>, Dispos
             //   * example is sandbox context created when login to sandbox first time
             // (3) there is namespace set in context and namespace exists in the cluster
             // (4) there is namespace set in context and namespace does not exist in the cluster
-            const pOrNs = await this.odo3.getNamespaces();
+            const namespaces = await this.odo3.getNamespaces();
```

Comment:
According to the comments above, we're logged into a cluster. Can we ensure `isLoggedIn` is set to `true` as I describe in https://github.com/redhat-developer/vscode-openshift-tools/issues/2848#issuecomment-1520843795  to fix the issue there also ?

Reply:
I think Muthu's fix here: https://github.com/redhat-developer/vscode-openshift-tools/pull/2851 makes more sense